#True if tracking is in progress
#False if not
bool tracking_flag
